contrib/sentry-native-cmake/CMakeLists.txt

```diff
@@ -28,11 +28,16 @@ set (SRCS
     ${SRC_DIR}/src/sentry_unix_pageallocator.c
     ${SRC_DIR}/src/path/sentry_path_unix.c
     ${SRC_DIR}/src/symbolizer/sentry_symbolizer_unix.c
-    ${SRC_DIR}/src/modulefinder/sentry_modulefinder_linux.c
     ${SRC_DIR}/src/transports/sentry_transport_curl.c
     ${SRC_DIR}/src/backends/sentry_backend_none.c
 )
 
+if(APPLE)
```

Comment:
Ok.

PS. Actually we use `sentry-native` library only as a strange way to send JSON over HTTPS. We don't need all this stuff. Maybe I will remove this library or stip it down later.